build/dependencies.props

```diff
@@ -1,6 +1,6 @@
-<Project>
+﻿<Project>
   <PropertyGroup>
-    <AspNetCoreLabsVersion>0.3.0-*</AspNetCoreLabsVersion>
+    <AspNetIntegrationTestingVersion>0.4.0-*</AspNetIntegrationTestingVersion>
```

Comment:
To avoid breaking the world when we introduced breaking changes: https://github.com/aspnet/Hosting/pull/996#issuecomment-290195195